how-to/how-to-use-kata-containers-with-acrn.md

```diff
@@ -0,0 +1,108 @@
+# Kata Containers with ACRN
+
+This document provides an overview on how to run Kata containers with ACRN hypervisor and device model.
+
+- [Introduction](#introduction)
+- [Pre-requisites](#pre-requisites)
+- [Install and Configure Docker](#install-and-configure-docker)
+- [Configure Kata Containers with ACRN](#configure-kata-containers-with-acrn)
+
+## Introduction
+
+ACRN is a flexible, lightweight Type-1 reference hypervisor built with real-time and safety-criticality in mind. It is optimized to streamline embedded development through an open source platform.
```

Comment:
Suggested rewrite for an active voice (second sentence): 
... ACRN uses an open source platform making it optimized to streamline embedded development.